oneview_redfish_toolkit/tests/api/test_blade_thermal.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
@danielreed this file would not have the name like **test_thermal.py** and test Thermal class using Enclosure, Blade and Rack? Overall it's a great PR, and I'll accept it as soon as this question to be resolved.